ledger/participant-integration-api/src/main/scala/platform/store/dao/events/EventsTableFlatEventsRangeQueries.scala

```diff
@@ -159,6 +157,7 @@ private[events] object EventsTableFlatEventsRangeQueries {
       QueryParts.ByArith(
         read = (range, limitOpt) => {
           val limitClause = limitOpt.map(sqlFunctions.limitClause).getOrElse("")
+
```

Comment:
@danielporterda : [nit] avoid  whitespace / newline changes to EventsTableFlatEventsRangeQueries.scala just to make it clear nothing functionally being touched here (I believe all changes to the file are whitespace)

Reply:
yes, i apparently left in a newline for every printline i included 🤦. removing